src/test/java/org/zalando/nakadi/controller/SubscriptionControllerTest.java

```diff
@@ -176,8 +194,33 @@ public void whenGetNoneExistingSubscriptionThenNotFound() throws Exception {
                 .andExpect(content().string(jsonHelper.matchesObject(expectedProblem)));
     }
 
+    @Test
+    public void whenPostSubscriptionThenWithNoReadScope() throws Exception {
+        when(eventTypeRepository.findByName("myET")).thenReturn(EventTypeTestBuilder.builder()
+                .name("myET")
+                .readScopes(Collections.singleton("oauth.read.scope"))
+                .build());
+
+        expectedException.expectCause(IsInstanceOf.instanceOf(IllegalScopeException.class));
+        expectedException.expectMessage("Client has to have scopes: [oauth.read.scope]");
```

Comment:
Why don't you just check which problem we have in response? Like here for example https://github.com/zalando/nakadi/pull/373/files#diff-8cb6844b0c2bfa55bfd6a5e918d7e925L176